packages/ketcher-core/src/application/editor/Editor.ts

```diff
@@ -145,6 +145,7 @@ interface IAutochainMonomerAddResult {
 
 export const EditorClassName = 'Ketcher-polymer-editor-root';
 export const KETCHER_MACROMOLECULES_ROOT_NODE_SELECTOR = `.${EditorClassName}`;
+const NATURAL_AMINO_ACID_MODIFICATION_TYPE = 'Natural amino acid';
```

Comment:
We have the same constant defined at `packages/ketcher-macromolecules/src/components/contextMenu/SelectedMonomersContextMenu/helpers.ts`, perhaps we could have the single source of truth for this value because it seems to be substantial?